firmware/gameport-adapter/CH-f16-combat-stick.h

```diff
@@ -0,0 +1,88 @@
+// This file is part of Necroware's GamePort adapter firmware.
+// Copyright (C) 2021 Necroware
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+//
+// CH F16 Combat Stick from 1995
+// Made by Stefan Geissler
+//  3 axes
+// 10 buttons
+//  1 Hat with 4 positions
+// The 4th axis (index 2) is ignored, because there is a big jitter
```

Comment:
This is interesting information, but it would be better placed where the data is been read to be better visible, in the `update()` function.